src/control/server/ctl_harness.go

```diff
@@ -35,8 +35,14 @@ import (
 	"github.com/daos-stack/daos/src/control/system"
 )
 
-// defaultTimeout used in harness requests.
-const defaultTimeout = 10 * time.Second
+const (
+	// timeout per rank request
+	harnessClientRankTimeount    = 3 * time.Second
+	harnessClientStopRankTimeout = 6 * time.Second
+	// timeout per harness request
+	harnessClientTimeount    = maxIoServers * harnessClientRankTimeount
```

Comment:
I'm a little wary of having all of these similarly-named timeouts. At least for me, it's hard to keep track of what's what later on in the code without constantly going back to the definitions to refresh my short-term memory.

I'd suggest something more like:
```go
// rank request handler timeouts
rankRequestTimeout = 3 * time.Second
rankStopTimeout = rankRequestTimeout * 2
// client-side timeouts for requests sent to remote harnesses
clientRequestTimeout = maxIoServers * rankRequestTimeout
clientStopTimeout = maxIoServers * rankStopTimeout
```